tiptapy/__init__.py

```diff
@@ -165,7 +176,9 @@ def register_renderer(cls):
 def convert_any(in_data):
```

Comment:
Hey ! @shon can this be converted to just an old version. 

```python 
def convert_any(in_data):
    typ = in_data.get("type")
    renderer = renderers.get(typ)
    return renderer.render(in_data)
```

As for type `doc`'s `is_renderable(in_data)` would always return `True` so this line might be redundant and end `return ''` won't be returned ever.